tests/bears/DebugBearsTest.py

```diff
@@ -50,17 +50,17 @@ def test_run_return_yield_with_debugger(self):
         self.assertEqual(lines[5], '-> yield 3')
 
     def test_run_return_list_with_debugger(self):
-        args = ('a', ('b', 'c'))
-        kwargs = {'d': 'e'}
-        result, output = execute_debugger('qc', func2, *args, **kwargs)
+        args = ()
+        kwargs = {}
```

Comment:
Imo it's okay if you completely omit those now. `execute_debugger` is designed to optionally accept keyword-params and positional params, everybody understands what it means if you don't pass any arguments at all ;)
```
execute_debugger('q', func2)
```